model/swagger.yml

```diff
@@ -37,14 +61,218 @@ paths:
           $ref: "#/components/responses/Error"
       parameters:
         - $ref: "#/components/parameters/Fields"
-        - $ref: "#/components/parameters/Group"
+        - $ref: "#/components/parameters/Class"
         - $ref: "#/components/parameters/Keyword"
         - $ref: "#/components/parameters/Limit"
         - $ref: "#/components/parameters/Query"
         - $ref: "#/components/parameters/Sort"
         - $ref: "#/components/parameters/Start"
+  /classes/{class}/{identifier}/members:
```

Comment:
@al-niessner do we want to support `/classes/{class}/{identifier}` as well?

Reply:
@jordanpadams 

I can add it if you like. Just out of curiosity, how is this different than /products/{identifier}?